<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindowClass</class>
 <widget class="QMainWindow" name="LoginWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="centralLayout" stretch="1,0">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="ptEdit_picture"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vLayout_loginInfo">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="maximumSize">
           <size>
            <width>15000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="ptEdit_accountIcon"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_loginInfo">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="7" column="0">
            <widget class="QCheckBox" name="cBox_LoginAuto">
             <property name="text">
              <string>自动登录</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QGroupBox" name="gBox_loginRegist">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="pbtn_login">
                <property name="text">
                 <string>登录</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbtn_regist">
                <property name="text">
                 <string>注册</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="cBox_savePassword">
             <property name="text">
              <string>保存密码</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_password">
               <property name="text">
                <string>密码：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lEdit_password"/>
             </item>
            </layout>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_accountPassError">
             <property name="text">
              <string>账户或密码错误</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_account">
               <property name="text">
                <string>账户：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ledit_account"/>
             </item>
            </layout>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="cBox_languageChange">
             <item>
              <property name="text">
               <string>中文简体（CHN）</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>美国英语（USA）</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LoginWindow.qrc"/>
 </resources>
 <connections/>
</ui>
